### PR TITLE
[DXP-739] Setup EULA for streamx, quasar-reactive-messaging and streamx-cli repositories

### PR DESCRIPTION
**Pull request by marekczajkowski**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1 +1,54 @@
...
+- make amendments to the source code of the Software
+- use the Software for any external, including commercial, purposes including placing it or products integrated with it on the market,
+- make the Software available to third parties, copy the Software for publication or distribution,
+- use the Software to create separate works, except for copies of the works created for trial purposes.
+
+This EULA can be revoked by DS at any time. In such case you are obliged to immediately cease the use of the Software and permanently remove all its components from your infrastructure and all data storage devices.
+
+Privacy
+
+The Software will not communicate with DS servers therefore no personal data will be processed by us.
```
Comment (jgardo-ds) [inline LICENSE.txt:38]: In website there is such text in this section:
```
The Software may communicate with DS servers only to receive and install updates, and to send anonymized usage information including (IP) address, and the date and time the Software was used, therefore no personal data will be processed by us.```

File: LICENSE.txt [hunk 2]
```diff
@@ -1 +1,54 @@
-DS EULA License
\ No newline at end of file
+StreamX End-User License Agreement
+Version 1.0, April 2024
```
Comment (jgardo-ds) [inline LICENSE.txt:2]: On website this fragment do not exist.
```
Version 1.0, April 2024
https://www.streamx.dev/licenses/eula-v1-0.html
```

**Review by jgardo-ds — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 2 above)
Comment (marekczajkowski) [reply LICENSE.txt:2]: its fine

File: LICENSE.txt (hunk 1 above)
Comment (marekczajkowski) [reply LICENSE.txt]: changed

**Review by jgardo-ds — APPROVED** [review]

**Pull request merged**